samcli/lib/providers/sam_api_provider.py

```diff
@@ -98,6 +115,100 @@ def _extract_from_serverless_api(
         collector.stage_variables = stage_variables
         collector.cors = cors
 
+        auth = properties.get(SamApiProvider._AUTH, {})
+        if auth:
+            if auth.get(SamApiProvider._DEFAULT_AUTHORIZER):
+                collector.set_default_authorizer(logical_id, auth.get(SamApiProvider._DEFAULT_AUTHORIZER))
+
+            self._extract_authorizers_from_props(logical_id, auth, collector, Route.API)
+
+    @staticmethod
+    def _extract_authorizers_from_props(logical_id: str, auth: dict, collector: ApiCollector, event_type: str) -> None:
+        """
+        Extracts Authorizers from the Auth properties section of Serverless resources
+
+        Parameters
+        ----------
+        logical_id: str
+            The logical ID of the Serverless resource
+        auth: dict
+            The Auth property dictionary
+        collector: ApiCollector
+            The Api Collector to send the Authorizers to
+        event_type: str
+            What kind of API this is (API, HTTP API)
+        """
+        API_IDENTITY_SOURCE_PREFIX = "method."
+        HTTP_IDENTITY_SOURCE_PREFIX = "$"
+
+        prefix = API_IDENTITY_SOURCE_PREFIX if event_type == Route.API else HTTP_IDENTITY_SOURCE_PREFIX
+        authorizers: Dict[str, Authorizer] = {}
+
+        for auth_name, auth_props in auth.get(SamApiProvider._AUTHORIZERS, {}).items():
+            authorizer_type = auth_props.get(SamApiProvider._FUNCTION_TYPE, LambdaAuthorizer.TOKEN).lower()
+            identity_object = auth_props.get(SamApiProvider._IDENTITY, {})
+
+            function_arn = auth_props.get(SamApiProvider._FUNCTION_ARN)
+
+            if not function_arn:
+                LOG.info("Authorizer '%s' is currently unsupported (must be a Lambda Authorizer), skipping", auth_name)
```

Comment:
Is it appropriate to change the log statement from an info log to a debug log? My concern is that if the project has multiple Lambda functions, the log output could become overwhelming.

Reply:
Yeah, the same concern/questions were brought up in a previously merged in PR about too many/too few log statements. One idea we could do, is to change these over to debug statements and add some logic to just do a single info log at the end to let them know there are unsupported authorizers in the template.

Reply:
Opted to change some of the LOGs (leaving the one about not being to parse ARN) to debugs. Since we also have an authorizer linking step, the user will be alerted there if someone is wrong, so debug output in this PR is suitable.